### PR TITLE
Improve description text when changing Key File credential

### PR DESCRIPTION
**Pull request by mmokrejs**
The current text and the three buttons available are confusing.
It is unclear whether one should Browse current master key file
or type in a new-one and click Generate ... Make it clear what
happens.

Ideally, the "Key File" could be replaced with "Master Key File"
and the path on the "Browse" line should show in grey path to the
current keyfile.

[NOTE]: # ( Describe your changes in detail, why is this change required? )
[NOTE]: # ( Explain large or complex code modifications. )
[NOTE]: # ( If it fixes an open issue, please add "Fixes #XXX" )

## Screenshots
[TIP]:  # ( Do not include screenshots of your actual database! )

## Testing strategy
[NOTE]: # ( Please describe in detail how you tested your changes. )
[TIP]:  # ( We expect new code to be covered by unit tests and documented with doc blocks! )

## Type of change
[NOTE]: # ( Please remove all lines which don't apply. )
- ✅ Bug fix (non-breaking change that fixes an issue)
- ✅ New feature (change that adds functionality)
- ✅ Breaking change (causes existing functionality to change)
- ✅ Refactor (significant modification to existing code)
- ✅ Documentation (non-code change)

### CODE REVIEW

**Comment by mmokrejs** [comment]
The proposed change shows here in the GUI:

keepassxc_window1

keepassxc_window2

The change does not change the translations files. I will leave it up to maintainers. Just wanted to propose concrete main text change.

**Comment by droidmonkey** [comment]
That is too much text. I was thinking to simply add before the text field something like: Press Generate to make a new key file, or select and existing one with Browse.

**Comment by mmokrejs** [comment]
I agree it seems too much text but ... something should happen.

The window shoudl show where is teh current master keyfile placed, like the very first window for each database shows.

The Browse button should not motivate confused user to browse to that file again. Overwriting existing keyfile is risky if something happens meanwhile. This should be clear if user sees what is the curretnly used master keyfile, if any.

This "window" should actually allow users having no separate master keyfle to get one, right?

It is interesting where the fixed size 128 byte size comes from, is that configurable? ;-)

Probably making a radio button for Generate and Browse lines would make things clear.

I was really puzzled what does this ask me to do. Still, at the very top should be shown whether currently there si any separate master keyfile or not. And educating users to have a separate file not copied to any cloud (unlike the databse file) would be advantegous. I think this should be a default, a sane a secure proposal.

**Comment by droidmonkey** [comment]
I greatly simplified the text and organized the widget better:
image

**Review by droidmonkey — APPROVED** [review]

**Comment by mmokrejs** [comment]
Thank you for your efforts. "Master Key File" would sound much clearer to me but I can live with that.

**Pull request merged**
